weave-js/src/components/RadixTooltip/Tooltip.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
+
+import * as RadixTooltip from './index';
+
+export type TooltipProps = {
+  trigger: React.ReactNode;
+  content: React.ReactNode | string;
+  /**
+   * This determines if the trigger and content components should be wrapped in
+   * RadixTooltip.Trigger and RadixTooltip.Content, respectively. Otherwise, it
+   * will just assume props passed in are properly wrapped in RadixTooltip anatomy.
+   * Having this option allows a bit more flexibilty and less prop passing.
+   *
+   * Setting it to false will tell the component to not apply the RadixTooltip.Trigger
+   * and RadixTooltip.Content around the props.
+   */
+  nowrap?: boolean;
+};
+
+/**
+ * This is a more ergonomic Tooltip built on top of our RadixTooltip. If this component does not
+ * meet your needs, please refrain from modifying this and consider using RadixTooltip directly.
+ */
+export const Tooltip = ({trigger, content, nowrap = false}: TooltipProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  if (nowrap) {
+    <RadixTooltip.Root open={isOpen} onOpenChange={setIsOpen}>
+      {trigger}
+      <RadixTooltip.Portal>{content}</RadixTooltip.Portal>
```

Comment:
**thought:** do you think `nowrap` should include the Portal too?

Reply:
i went with another prop because the trigger and content wrappers are critical for the radix tooltip to work apparently. I tried without the trigger and content wrappers, and tooltip content would not show up.